src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketDiscoverableHeadType.java

```diff
@@ -0,0 +1,6 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+public enum BitbucketDiscoverableHeadType {
+
+    PULL_REQUEST
```

Comment:
We can eventually add `BRANCH` here once we're ready to move branch discovery away from the `GitSCMSource` and into to the "new" implementation.